### PR TITLE
Jennifer/issue 563

### PR DESCRIPTION
**Pull request by jaube05**

### CODE REVIEW

File: _guide/semantic-html.md [hunk 1]
```diff
@@ -11,7 +11,11 @@ roles:
 - Developer
 ---
 
+<<<<<<< HEAD
 Writing semantic HTML can make your site more accessible. When you switch your browser to "read mode", only the main content will be visible. Read mode hides:
+=======
+Writing semantic HTML can make your site more accessible. When you switch your browser to "read mode", only the main content will be visible. Read mode hides:
+>>>>>>> 11f717f (adding page to guide for semantic html)
```
Comment (dmundra) [inline _guide/semantic-html.md:18]: @jaube05 looks like we got some merge commit code added.I think it just needs manual clearing up.

File: _data/guide.yml [hunk 2]
```diff
@@ -64,7 +64,7 @@ docs:
 - title: Semantic HTML
   url: /semantic-html
   description:
-  icon: <i class="far fa-file-code fa-5x" aria-hidden="true"></i>
+  icon: <i class="far fa-code fa-5x" aria-hidden="true"></i>
```
Comment (dmundra) [inline _data/guide.yml:67]: @jaube05 this one came back and unfortunately it is still not in the fa collection.

**Review by dmundra — CHANGES_REQUESTED** [review]
Other than the merge issue and older change the logo itself is looking great. Thank you for adding it!

File: _guide/semantic-html.md (hunk 1 above)
Comment (jaube05) [reply _guide/semantic-html.md:18]: Fixed

**Review by dmundra — CHANGES_REQUESTED** [review]
Sorry, @jaube05 last change is the font-awesome icon. `fa-code` is still coming up empty on the QA site.

File: _data/guide.yml (hunk 2 above)
Comment (jaube05) [reply _data/guide.yml:67]: fixing

**Review by dmundra — APPROVED** [review]
Thank you @jaube05! Approved!

**Pull request merged**